### DISCUSSION TITLE
프리트레인 모델 불러오는 부분

### DISCUSSION THREAD

**Discussion by ppskj178** [boostcampaitech2/object-detection-level2-cv-06#78] [Q&A] [answered]
@kmouleejunhyuk @jaehwan-AI 

저는 자꾸 pth가 안맞는다는식으로 에러가 나오는데...

num_class를 수정하면 백본 외에는 못불러오는게 맞지않나요?

`init_weights = True` 이 옵션을 해주면 되는건가 싶어서...

지금 작업 돌려놓고 백신 맞으러가면서 질문 드려봅니다...

**Comment by kmouleejunhyuk** [comment]
@ppskj178  멘토님께 문의 결과 pth의 head에 class num 이 안맞으면 그냥 그 부분은 초기화하고 진행한다고 합니다. pth가 안맞는다는 식으로 에러가 난다면 잘못된 pth를 사용하시는게 아닌지 의심스럽네요

**Comment by jaehwan-AI** [comment]
weight가 정확하게 일치하지 않는다는 warning이 찍히지만 log에 보면 자동으로 초기화해서 진행하는 것을 확인했습니다.
준혁님 말씀대로 잘못된 pth가 input된 것 같습니다.

**Comment by jiwoo0212** [comment, marked as answer]
아니면 pth의 절대경로를 입력해요! 저도 같은 오류가 났었는데 절대경로 입력하니까 해결됐던 것 같습니다

**Reply by ppskj178** [reply]
제정신인가 싶네요.... 계속 이거저거 해봤는데.. 절대경로하니까 한방에됬습니다..;;
